metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -819,7 +819,18 @@ class MetalsLanguageServer(
           if (userConfig.symbolPrefixes != old.symbolPrefixes) {
             compilers.restartAll()
           }
-          if (userConfig.pantsTargets != old.pantsTargets) {
+          if (userConfig.bloopVersion != old.bloopVersion) {
+            languageClient
+              .showMessageRequest(
+                Messages.BloopVersionChange.params(userConfig.bloopVersion)
+              )
+              .asScala
+              .flatMap {
+                case item if item == Messages.BloopVersionChange.reconnect =>
```

Comment:
This will crash with `MatchError` when the user doesn't want to reconnect.

Reply:
Thanks, fixed!